<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RTCompare</class>
 <widget class="qMRMLWidget" name="RTCompare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>817</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <property name="collapsedHeight">
      <number>0</number>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Accepted Study Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Accepted Modality</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="tbStudyDesc">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="tbModality"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="initBtn">
        <property name="text">
         <string>Initialize</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Config JSON</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="ctkPathLineEdit" name="patientsCSVPath">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="showHistoryButton">
         <bool>false</bool>
        </property>
        <property name="nameFilters">
         <stringlist>
          <string>*.csv</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="initConfigsBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Initialize configs</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="ctkPathLineEdit" name="resultsCSVPath">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="showHistoryButton">
         <bool>false</bool>
        </property>
        <property name="nameFilters">
         <stringlist>
          <string>*.csv</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QTableWidget" name="structsTbl">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkDirectoryButton" name="outputDirBtn">
        <property name="directory">
         <string>.</string>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="ctkPathLineEdit" name="jsonPath">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="showHistoryButton">
         <bool>false</bool>
        </property>
        <property name="nameFilters">
         <stringlist>
          <string>*.json</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Patients CSV</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Patients</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QTableWidget" name="patientsTbl">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Output folder</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Structures</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkDirectoryButton" name="sourceDirBtn">
        <property name="elideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Results CSV</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Source folder</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="btnExtractStructureInfo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Extract structure info</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="runBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Start processing</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
